erpnext/stock/doctype/stock_ledger_entry/stock_ledger_entry.py

```diff
@@ -38,6 +38,11 @@ def on_submit(self):
 		self.check_stock_frozen_date()
 		self.actual_amt_check()
 
+		if self.batch_no:
+			batch = frappe.get_doc("Batch", self.batch_no)
+			batch.calculate_batch_qty()
+			batch.save()
```

Comment:
This may lead to performance issue, directly you can update a field via sql query or you can fetch this information in runtime with ajax